stylesheets/scss/_mixins.scss

```diff
@@ -38,9 +38,15 @@
   } // ie8
 } // @mixin responsive_image
 
+//rem fallback.  Usage: @include rem(font-size, 1.4);
+@mixin rem($property, $rem) {
+  #{$property}: #{$rem*10}px;
+  #{$property}: #{$rem}rem;
+}
+
+//The old rem fallback
 @mixin font-size($size) {
```

Comment:
This was nice of you to keep this for legacy. Let's maybe give it 6 months then nuke it.